src/main/scala/util/ClockDivider.scala

```diff
@@ -5,15 +5,11 @@ package util
 import Chisel._
 
 /** Divide the clock by 2 */
```

Comment:
Can you add a note here similar to the one in the Verilog, in case people don't go read the Verilog... and that this is intended to be used for simulation or with careful application of constraints.